From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: xiaoyinl <xiaoyinl@users.noreply.github.com>
Date: Fri, 14 Jun 2024 14:40:22 +0800
Subject: [PATCH] 6637: Update InstallationGuide.html to remove instructions
 for Windows 7

Since Windows 7 is not supported, there's no need to keep the instructions about how to set Environment Variables on Windows 7.
---
 GhidraDocs/InstallationGuide.html | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/GhidraDocs/InstallationGuide.html b/GhidraDocs/InstallationGuide.html
index 6f55051ea3..d35f642650 100644
--- a/GhidraDocs/InstallationGuide.html
+++ b/GhidraDocs/InstallationGuide.html
@@ -180,11 +180,7 @@ destination using any unzip program (built-in OS utilities, 7-Zip, WinZip, WinRA
             <ol>
               <li>
                 <p>
-                  <i>Windows 10:</i> Right-click on Windows start button, and click <b>System</b>
-                </p>
-                <p>
-                  <i>Windows 7:</i> Click Windows start button, right-click on Computer, and click 
-                  <b>Properties</b>
+                  <i>Windows 10, 11:</i> Right-click on Windows start button, and click <b>System</b>
                 </p>
               </li>
               <li>Click <b>Advanced system settings</b></li>
-- 
2.45.1

